services/repository/src/main/java/org/sagebionetworks/auth/controller/OpenIDConnectController.java

```diff
@@ -338,7 +338,7 @@ OIDCTokenResponse getTokenResponse(
 	@RequestMapping(value = UrlHelpers.OAUTH_2_USER_INFO, method = {RequestMethod.GET})
 	public @ResponseBody
 	Object getUserInfoGET(
-			@RequestHeader(value = AuthorizationConstants.SYNAPSE_AUTHORIZATION_HEADER_NAME, required=true) String authorizationHeader,
+			@RequestHeader(value = AuthorizationConstants.SYNAPSE_AUTHORIZATION_HEADER_NAME, required=false) String authorizationHeader,
```

Comment:
We might want to handle the proper exception thrown by spring in the BaseControllerExceptionHandlerAdvice (e.g. might be a MissingRequestHeaderException) instead.